apps/web/src/server/applications/common/validators/dates.validators.js

```diff
@@ -35,26 +52,63 @@ export const createValidationDateValid = (
 		})
 		.withMessage(`Enter a valid year for the ${extendedFieldName}`)
 		.custom(() => {
-			return dateIsValid(year, month, day);
+			return (
+				!(stringDay.length !== 2 || stringMonth.length !== 2 || stringYear.length !== 4) &&
+				dateIsValid(year, month, day)
+			);
 		})
-		.withMessage(`Enter a valid ${extendedFieldName}`);
+		.withMessage(`You must enter the ${extendedFieldName} in the correct format`);
 };
 
 /**
  *
- * @param {string} fieldName
- * @param {string} extendedFieldName
- * @param {{day: string, month: string, year: string}} dateParams
+ * @param {{fieldName: string, extendedFieldName: string}} field
+ * @param {{fieldName: string, extendedFieldName: string}|null} compareField
+ * @param {Record<string, string>} data
+ * @returns {import("express-validator").ValidationChain}
+ */
+export const validationDateStartBeforeEnd = (field, compareField, data) => {
+	const { fieldName, extendedFieldName } = field;
+
+	const stringDay = data[`${fieldName}.day`] || '';
+	const stringMonth = data[`${fieldName}.month`] || '';
+	const stringYear = data[`${fieldName}.year`] || '';
+
+	const stringCompareDay = data[`${compareField?.fieldName}.day`] || '32';
+	const stringCompareMonth = data[`${compareField?.fieldName}.month`] || '13';
+	const stringCompareYear = data[`${compareField?.fieldName}.year`] || '9999';
+
+	return body(fieldName)
+		.custom(() => {
+			if (stringYear > stringCompareYear) {
+				return false;
+			}
+			if (stringMonth > stringCompareMonth) {
+				return false;
+			}
+			if (stringDay > stringCompareDay) {
+				return false;
+			}
+			return true;
```

Comment:
If user prompts the same date (or time) in the start and end dates, no error is prompted.

startDate > endDate => error
startDate = endDate => _**no error**_
startDate < endDate => no error